dronegen/tag.go

```diff
@@ -148,6 +176,12 @@ func tagCopyArtifactCommands(b buildType) []string {
 		}
 	}
 
+	if b.hasTeleportConnect() {
+		commands = append(commands,
+			`cp /go/src/github.com/gravitational/webapps/packages/teleterm/build/release/teleconnect*.{tar.gz,deb,rpm} /go/artifacts/`,
```

Comment:
We just changed `teleconnect` to `teleport-connect` in webapps.
```suggestion
			`cp /go/src/github.com/gravitational/webapps/packages/teleterm/build/release/teleport-connect*.{tar.gz,deb,rpm} /go/artifacts/`,

Reply:
Does this affect the Mac builds too?

Reply:
No, this change affects only Linux packages.